homework/fibonacci/fibonacci.hpp

```diff
@@ -1,11 +1,18 @@
 #pragma once
 
 int fibonacci_iterative(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
+    int a = 0;
+    int b = 1;
```

Comment:
What is 'a? Variable should always represent real operation.
Like `result` `divider` `multiplier` `error_code` etc...